### PR TITLE
Make Mantis mask file extension independent from segmentation file extension

### LINKED ISSUE #862: `create_mantis_dir` assumes segmentation overlay extension is the same as the pixel/cell overlay extension
*Please refer to our [FAQ](https://ark-analysis.readthedocs.io/en/latest/_rtd/faq.html) and look at our [known issues](https://github.com/angelolab/ark-analysis/issues) before opening a bug report.*

**Describe the bug**
In the `create_mantis_dir` function, it assumes that the segmentation extension is the same as the extension of the overlay mask:
https://github.com/angelolab/ark-analysis/blob/de39b62b1c310a3f63549b2238c8e72af9ece56f/ark/utils/plot_utils.py#L504

This causes errors when the segmentation mask suffix and overlay mask suffix are different. In my testing, I am using segmentation masks that end in ".tif", but in the pixel clustering notebook, the default is to generate the pixel masks using ".tiff."

**Expected behavior**
Be able to read ".tif" or ".tiff."

**To Reproduce**
image

### PR DESCRIPTION
**Pull request by alex-l-kong**
**What is the purpose of this PR?**

Closes #862. Currently it is assumed that the segmentation and Mantis mask file extensions are the same, which will not hold if segmentation masks use `.tif`.

**How did you implement your changes**

We have to hardcode the `.tiff` extension for Mantis masks as opposed to using `seg_suffix_ext`.

### CODE REVIEW

**Comment by alex-l-kong** [comment]
@cliu72 changed the file path, let me know if there's anything else that needs changing here.

**Comment by cliu72** [comment]
Looking this over, it looks like `seg_suffix_ext` is also being used to copy the single-channel images into the mantis directory, which I think is confusing because the single-channel images don't necessarily have the same extension as the segmentation file.
https://github.com/angelolab/ark-analysis/blob/a45145dab7861a68f591ed80f610d3bc79ce5f2e/ark/utils/plot_utils.py#L499

As I talk about in this issue https://github.com/angelolab/ark-analysis/issues/866, `load_imgs_from_tree` looks for images with ['.tiff', '.jpg', '.png'], and I asked in that issue to include 'tif' in that list as well. Here, I think we can use the same logic as `load_imgs_from_tree` to decide which channels to copy over in `create_mantis_dir`.

So basically, change those lines in `create_mantis_dir` to something like this:
```
            # copy all channels into new folder
            chans = io_utils.list_files(img_source_dir, substrs=substrs=['.tiff', '.tif','.jpg', '.png'] )
            for chan in chans:
                shutil.copy(os.path.join(img_source_dir, chan), os.path.join(output_dir, chan))
```

I think we can then delete the lines used to get `seg_suffix_ext` (won't be necessary to get the extension anymore):
https://github.com/angelolab/ark-analysis/blob/a45145dab7861a68f591ed80f610d3bc79ce5f2e/ark/utils/plot_utils.py#L452-L455

**Comment by alex-l-kong** [comment]
@cliu72 `seg_suffix_ext` references have been removed.

**Review by cliu72 — APPROVED** [review]
Looks good

**Comment by alex-l-kong** [comment]
@ngreenwald can you please help merge this one in, a minor coveralls issue is preventing that from happening.

**Review by ngreenwald — APPROVED** [review]
Looks good
